src/app/pages/form/form.component.ts

```diff
@@ -1,9 +1,41 @@
 import { Component } from '@angular/core';
+import { FormBuilder, Validators } from '@angular/forms';
+
+interface City {
+  name: string;
+  code: string;
+}
 
 @Component({
-    selector: 'app-form',
-    templateUrl: './form.component.html'
+  selector: 'app-form',
+  templateUrl: './form.component.html',
+  styleUrls: ['./form.component.scss'],
 })
 export class FormComponent {
+  userForm = this.fb.group({
+    location: ['', Validators.required],
+    destination: [''],
+    selectedCity: [''],
+    departureDate: [''],
+    returnDate: [''],
+    passengers: [''],
+  });
+
+  cities: City[] = [
+    { name: 'New York, USA', code: 'JFK' },
```

Comment:
ogólnie pisanie takich stringów z palca to trochę zła praktyka i raczej się tego unika - dobrym sposobem są enumy i tłumaczenia :). Najpierw zaczniemy sobie od `code` :) 

1. W folderze `enums` stwórz enum `city-code.enum.ts`
2. jego kod powinien wyglądać w ten sposób 

```
export enum CityCode{
    newYork = 'JFK',
    rome = 'FCO'
 //Tutaj pozostałe kody lotnisk
}
```

3. W interface `City` zamieniach `code:string` na `code: CityCode` -> wtedy oznaczamy że każdy `code` musi być wartością z enuma `CityCode`
4. W arrayu cities musimy teraz pozamieniać obiekty :) 
Zamiast `{ name: 'New York, USA', code: 'JFK' },` będziesz  musiała umieścić `{ name: 'New York, USA', code: CityCode.newYork },` i tak dla wszystkich obiektów ich dedykowany kod :)